docs/messaging/ios-notification-images.md

```diff
@@ -33,6 +33,7 @@ Ensure that your new extension has access to Firebase/Messaging pod by adding it
 
 ```Ruby
 target 'ImageNotification' do
+  use_frameworks! :linkage => :static 
   pod 'Firebase/Messaging', '~> VERSION_NUMBER' # eg 6.31.0
```

Comment:
I think it's even more correct (in our context) to eliminate the version number part
That way the version number defers to whatever is set in other areas of the app which *should* be by `@react-native-firebase/app` but may be overridden (if really necessary) above in the file, but this way you don't have the version number scattered around in multiple places to maybe go out of date

```suggestion
  pod 'Firebase/Messaging'
```